src/wp-admin/network/themes.php

```diff
@@ -309,10 +405,36 @@
 		$message = _n( '%s theme deleted.', '%s themes deleted.', $deleted );
 	}
 	echo '<div id="message" class="updated notice is-dismissible"><p>' . sprintf( $message, number_format_i18n( $deleted ) ) . '</p></div>';
+} elseif ( isset( $_GET['enabled-auto-update'] ) ) {
+	$enabled = absint( $_GET['enabled-auto-update'] );
+	if ( 1 == $enabled ) {
```

Comment:
Following up from https://github.com/WordPress/wordpress-develop/pull/279#discussion_r426841452

This should be `if ( 1 === $enabled ) {}` and the same for `$disabled` below